bgpd/bgp_route.c

```diff
@@ -10379,7 +10380,11 @@ DEFUN (show_ip_bgp_large_community_list,
 
 	bool uj = use_json(argc, argv);
 
-	struct bgp *bgp = bgp_lookup_by_name(vrf);
+	/* without vrf/view keyword or 'vrf default' */
```

Comment:
bgp_vty_find_and_parse_afi_safi?

Reply:
Thanks for the review. I will re-work the code.